university-agent/src/main/java/nl/quintor/studybits/LedgerSeeder.java

```diff
@@ -89,7 +84,50 @@ public void seed(ContextRefreshedEvent event) throws InterruptedException, Execu
         }
     }
 
-    private boolean needsSeeding() {
+    public static void onboardIssuer(TrustAnchor steward, Issuer newcomer) throws InterruptedException, ExecutionException, IndyException, IOException {
+        // Create Codecs to facilitate encryption/decryption
+        MessageEnvelopeCodec stewardCodec = new MessageEnvelopeCodec(steward);
+        MessageEnvelopeCodec newcomerCodec = new MessageEnvelopeCodec(newcomer);
+
+        // Connecting newcomer with Steward
+
+        // #Step 4.1.2 & 4.1.3
```

Comment:
Does this method need the step comments (used in quindy)?

Reply:
Removed